### PR TITLE
WebDriver conformance changes for Firefox 110

### PR DESCRIPTION
**Pull request by whimboo**
This includes all the WebDriver conformance changes for Firefox 110 based on the [query on Bugzilla](https://bugzilla.mozilla.org/buglist.cgi?status_whiteboard=%5Bwebdriver%3Arelnote%5D&resolution=FIXED&query_format=advanced&v1=fixed&j_top=OR&status_whiteboard_type=substring&o1=equals&f1=cf_status_firefox110&list_id=16405065).

@juliandescottes can you please review? Thanks.

### CODE REVIEW

**Comment by juliandescottes** [comment]
Looks great, thanks @whimboo

**Comment by whimboo** [comment]
@Rumyra could you or someone else please review and merge? Thanks!

**Comment by bsmth** [comment]
I can take a look @whimboo 👀

File: files/en-us/mozilla/firefox/releases/110/index.md [hunk 1]
```diff
@@ -69,8 +69,22 @@ This article provides information about the changes in Firefox 110 that will aff
...
+
+- Added support for serialization and deserialization of generic platform objects ({{bug(1792524)}}), and for `NodeList` and `HTMLCollection` platform objects ({{bug(1802284)}}).
+
+- Added a `timestamp` field to the `browsingContext.domContentLoaded` and `browsingContext.load` events ({{bug(1790378)}}).
+
+- Added a `type` field to the response for `script.evaluate` and `script.callFunction` to indicate for `success` or `exception` ({{bug(1803599)}}).
+
 #### Marionette
 
+- The cache for known nodes (element and shadow root references) has been moved from the parent to the web content process to follow recent WebDriver classic changes ({{bug(1692468)}}).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/110/index.md:84]: minor nitpick

```suggestion
- The cache for known nodes (element and shadow root references) has been moved from the parent to the web content process following recent WebDriver classic changes ({{bug(1692468)}}).
```

**Comment by juliandescottes** [comment]
@bsmth Henrik is off this week, not sure if he will check this issue soon. If there are no news, I think it's fine to merge your suggestions and land it.

**Comment by bsmth** [comment]
> I think it's fine to merge your suggestions and land it.

Thank you @juliandescottes for the heads-up. In that case, I will apply the changes and merge shortly to make sure this goes live soon.

**Review by bsmth — APPROVED** [review]

**Pull request merged**